src/core/exporter.js

```diff
@@ -55,26 +55,29 @@ function cleanup(cloneDoc) {
   ).forEach(elem => elem.remove());
   body.classList.remove("toc-sidebar");
   removeReSpec(documentElement);
+
   const insertions = cloneDoc.createDocumentFragment();
+
   // Move meta viewport, as it controls the rendering on mobile.
   const metaViewport = cloneDoc.querySelector("meta[name='viewport']");
   if (metaViewport && head.firstChild !== metaViewport) {
     insertions.appendChild(metaViewport);
   }
+
   // Move charset to near top, as it needs to be in the first 512 bytes.
   let metaCharset = cloneDoc.querySelector(
     "meta[charset], meta[content*='charset=']"
   );
   if (!metaCharset) {
-    metaCharset = cloneDoc.createElement("meta");
-    metaCharset.setAttribute("charset", "utf-8");
+    metaCharset = hyperHTML`<meta charset="utf-8">`;
   }
   insertions.appendChild(metaCharset);
+
   // Add meta generator
-  const metaGenerator = cloneDoc.createElement("meta");
-  metaGenerator.name = "generator";
-  metaGenerator.content =
-    "ReSpec " + window.respecVersion || "Developer Channel";
+  const respecVersion = window.respecVersion || "Developer Channel";
+  const metaGenerator = hyperHTML`
+    <meta name="generator" content="${"ReSpec " + respecVersion}">`;
+
```

Comment:
Make sure you confirm this works... we I don't know what the owner document is here, and if adopting this node won't throw.

Reply:
I exported it using the ui-pill. It had the `meta generator` and others. I'll confirm again

Reply:
Update: works (exported html source: https://i.imgur.com/bKcE9To.png)

Reply:
The owner document would be `cloneDoc` as we are appending `meta` into `insertions` which gets inserted into `cloneDoc.head`. Right?